### PR TITLE
[bitnami/magento] Move upstreamed/magento to bitnami/magento

### PR DESCRIPTION
**Pull request by alemorcuq**
**Description of the change**

Moves the Magento chart to the Bitnami repo as it was deprecated upstream, and configure it to use the `bitnami/elasticsearch` chart.

**Benefits**

Be in control of all the dependencies in our charts.

**Additional information**

Work in progress.

**Checklist**
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

### CODE REVIEW

**Comment by carrodher** [comment]
Can you add and complete the following section about production configuration once finished?
```diff
--- a/bitnami/magento/README.md
+++ b/bitnami/magento/README.md
@@ -65,7 +65,7 @@ The following table lists the configurable parameters of the Magento chart and t
 | `image.repository`                   | Magento Image name                                                                   | `bitnami/magento`                                            |
 | `image.tag`                          | Magento Image tag                                                                    | `{TAG_NAME}`                                                 |
 | `image.debug`                        | Specify if debug values should be set                                                | `false`                                                      |
-| `image.pullPolicy`                   | Image pull policy                                                                    | `Always` if `imageTag` is `latest`, else `IfNotPresent`      |
+| `image.pullPolicy`                   | Image pull policy                                                                    | `IfNotPresent`                                               |
 | `image.pullSecrets`                  | Specify docker-registry secret names as an array                                     | `[]` (does not add image pull secrets to deployed pods)      |
 | `magentoHost`                        | Magento host to create application URLs                                              | `nil`                                                        |
 | `magentoLoadBalancerIP`              | `loadBalancerIP` for the magento Service                                             | `nil`                                                        |
@@ -174,6 +174,26 @@ $ helm install --name my-release -f values.yaml stable/magento

 > **Tip**: You can use the default [values.yaml](values.yaml)

+### Production configuration
+
+This chart includes a `values-production.yaml` file where you can find some parameters oriented to production configuration in comparison to the regular `values.yaml`.
+
+```console
+$ helm install --name my-release -f ./values-production.yaml bitnami/magento
+```
+
+- Use the Elasticsearch chart as search engine:
+```diff
+- elasticsearch.enabled: false
++ elasticsearch.enabled: true
+```
+
+- Start a side-car prometheus exporter:
+```diff
+- metrics.enabled: false
++ metrics.enabled: true
+```
+
 ### [Rolling VS Immutable tags](https://docs.bitnami.com/containers/how-to/understand-rolling-tags-containers/)

 It is strongly recommended to use immutable tags in a production environment. This ensures your deployment does not change automatically if the same tag is updated with a different image.
```

Mainly you need to do a diff between _values.yaml_ and _values-production.yaml_ and document in a diff way using the variable description present in the main table as the title of the item.

[Here](https://github.com/helm/charts/pulls?q=is%3Apr+Unify+and+document+production+values+is%3Aclosed) you can find more examples.

Also ensure that the output of the diff between _values.yaml_ and _values-production.yaml_ doesn't contain more elements than the desired ones, if so, please try to unify it.

File: bitnami/magento/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: magento
-version: 5.0.3
+version: 5.0.4
```
Comment (carrodher) [inline bitnami/magento/Chart.yaml]: Maybe we should increment the chart version in the `Chart.yml` in a minor, according to [semver](http://semver.org/):

> Given a version number `MAJOR.MINOR.PATCH`, increment the:
> - MAJOR version when you make incompatible API changes,
> - MINOR version when you add functionality in a backwards-compatible manner, and
> - PATCH version when you make backwards-compatible bug fixes.
> Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.

File: bitnami/magento/values.yaml [hunk 2]
```diff
@@ -158,12 +158,21 @@ mariadb:
...
   enabled: false
+  sysctlImage:
+    enabled: false
+  master:
+    replicas: 1
+  coordinating:
+    replicas: 1
+  data:
+    replicas: 1
+
```
Comment (carrodher) [inline bitnami/magento/values.yaml]: Unnecessary line

File: bitnami/magento/values-production.yaml [hunk 3]
```diff
@@ -155,15 +155,25 @@ mariadb:
       accessMode: ReadWriteOnce
       size: 8Gi
 
+  enabled: true
```
Comment (carrodher) [inline bitnami/magento/values-production.yaml]: Ensure that (if needed) it is present in both files

File: bitnami/magento/README.md [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+
+```console
+$ helm install bitnami/magento
+```
+
+## Introduction
+
+This chart bootstraps a [Magento](https://github.com/bitnami/bitnami-docker-magento) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+It also packages the [Bitnami MariaDB chart](https://github.com/kubernetes/charts/tree/master/stable/mariadb) which is required for bootstrapping a MariaDB deployment as a database for the Magento application.
```
Comment (tompizmor) [inline bitnami/magento/README.md]: the mariadb repo url is wrong. It should be the bitnami one

File: bitnami/magento/requirements.lock [hunk 5]
```diff
@@ -0,0 +1,9 @@
+dependencies:
+- name: mariadb
+  repository: https://kubernetes-charts.storage.googleapis.com/
```
Comment (tompizmor) [inline bitnami/magento/requirements.lock]: mariadb should also come from bitnami repo

File: bitnami/magento/requirements.yaml [hunk 6]
```diff
@@ -4,6 +4,6 @@ dependencies:
   repository: https://kubernetes-charts.storage.googleapis.com/
```
Comment (tompizmor) [inline bitnami/magento/requirements.yaml]: change it to bitnami repo

**Pull request merged**